build-conflicts.php

```diff
@@ -34,6 +34,11 @@
 use Roave\SecurityAdvisories\AdvisorySources\GetAdvisoriesFromGithubApi;
 use Roave\SecurityAdvisories\AdvisorySources\GetAdvisoriesFromMultipleSources;
 
+use function array_keys;
+use function count;
```

Comment:
```suggestion
use Psl\Iter;
```

$count = Iter\count($iterable);